infra/feast-operator/config/samples/v1alpha1_featurestore_postgres_db_volumes_ssl.yaml

```
@@ -0,0 +1,167 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-secret
+  namespace: default
+  labels:
+    app: postgres
+stringData:
+  POSTGRES_DB: mydatabase
+  POSTGRES_USER: admin
+  POSTGRES_PASSWORD: password
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: feast-data-stores
+  namespace: default
+stringData:
+  redis: |
+    connection_string: redis.feast.svc.cluster.local:6379
+  sql: |
+    path: postgresql+psycopg://admin:password@postgresql.default.svc.cluster.local:5432/mydatabase?sslmode=require&sslrootcert=/var/lib/postgresql/certs/ca.crt&sslcert=/var/lib/postgresql/certs/tls.crt&sslkey=/var/lib/postgresql/certs/tls.key
+    cache_ttl_seconds: 60
+    sqlalchemy_config_kwargs:
+        echo: false
+        pool_pre_ping: true
+  postgres: |
+    host: postgresql.default.svc.cluster.local
+    port: 5432
+    database: mydatabase
+    db_schema: public
+    user: admin
+    password: password
+    sslmode: require
+    sslkey_path: /var/lib/postgresql/certs/tls.key
+    sslcert_path: /var/lib/postgresql/certs/tls.crt
+    sslrootcert_path: /var/lib/postgresql/certs/ca.crt
+    vector_enabled: false
+    vector_len: 512
+  postgres_offline: |
+    host: postgresql.default.svc.cluster.local
+    port: 5432
+    database: mydatabase
+    db_schema: public
+    user: admin
+    password: password
+    sslmode: require
+    sslkey_path: /var/lib/postgresql/certs/tls.key
+    sslcert_path: /var/lib/postgresql/certs/tls.crt
+    sslrootcert_path: /var/lib/postgresql/certs/ca.crt
```

do we need separate configs for offline vs online? in the past i've used the same config for both feast service types.
```suggestion
  postgres: |
    host: postgresql.default.svc.cluster.local
    port: 5432
    database: mydatabase
    db_schema: public
    user: admin
    password: password
    sslmode: require
    sslkey_path: /var/lib/postgresql/certs/tls.key
    sslcert_path: /var/lib/postgresql/certs/tls.crt
    sslrootcert_path: /var/lib/postgresql/certs/ca.crt
    vector_enabled: false
    vector_len: 512
```

---

I have tested with the same configuration but it didn't work. seems like operator is expecting exact configuration and not ignoring if the secret is having extra properties which are not needed. in the case of offline we don't have below properties. 

```
    vector_enabled: false
    vector_len: 512
```

---

ok, but are those fields necessary for this example?